### PR TITLE
[feature](merge-cloud) support MoW table in cloud mode

### PR DESCRIPTION
**Pull request by liaoxin01**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by liaoxin01** [comment]
run buildall

**Comment by liaoxin01** [comment]
run buildall

**Comment by liaoxin01** [comment]
run buildall

**Comment by liaoxin01** [comment]
run buildall

**Comment by liaoxin01** [comment]
run buildall

**Comment by liaoxin01** [comment]
run buildall

**Comment by liaoxin01** [comment]
run buildall

**Comment by liaoxin01** [comment]
run buildall

**Comment by liaoxin01** [comment]
run buildall

**Comment by liaoxin01** [comment]
run buildall

**Comment by liaoxin01** [comment]
run buildall

**Review by dataroaring — DISMISSED** [review]
LGTM

**Comment by liaoxin01** [comment]
run buildall

**Comment by liaoxin01** [comment]
run feut

**Comment by liaoxin01** [comment]
run feut

File: be/src/cloud/cloud_engine_calc_delete_bitmap_task.cpp [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+            }
+
+            Status st = tablet->sync_rowsets();
+            if (!st.ok() && !st.is<ErrorCode::INVALID_TABLET_STATE>()) {
+                return st;
+            }
+            if (st.is<ErrorCode::INVALID_TABLET_STATE>()) [[unlikely]] {
+                add_succ_tablet_id(tablet->tablet_id());
+                LOG(INFO)
+                        << "tablet is under alter process, delete bitmap will be calculated later, "
```
[the remark this answers is not in the document]
Comment (liaoxin01) [reply be/src/cloud/cloud_engine_calc_delete_bitmap_task.cpp:86]: schema change and compaction will be submitted in next pr.

File: be/src/cloud/cloud_tablets_channel.cpp [hunk 2]
```diff
@@ -235,7 +235,16 @@ Status CloudTabletsChannel::close(LoadChannel* parent, const PTabletWriterAddBlo
         }
     }
 
-    // TODO(plat1ko): 6. set txn related delete bitmap if necessary
+    // 6. set txn related delete bitmap if necessary
```
[the remark this answers is not in the document]
Comment (liaoxin01) [reply be/src/cloud/cloud_tablets_channel.cpp:238]: Sync rowset is a heavy operation in cloud mode, we will consider whether to add it  later.

[the remark this answers is not in the document]
Comment (zhannngchen) [reply be/src/cloud/cloud_tablets_channel.cpp:238]: It's fine to add it later, or you can make some performance test on that.
In our performance optimization half a year ago, calculating delete bitmap for incremental rowsets before commit can significantly reduce the computational cost of the publish phase

**Review by zhannngchen — DISMISSED** [review]
LGTM

**Comment by liaoxin01** [comment]
run buildall

**Review by dataroaring — APPROVED** [review]
LGTM

**Review by zhannngchen — APPROVED** [review]

**Pull request merged**